PeerioTechnologies/peerio-desktop
# fix: Linux updater, remove "integrate with system" dialog

The dialog asking on first launch whether to "integrate AppImage with the system" -- that is, install .desktop file for it so it appears in the applications menu -- was confusing for users, and kinda annoying. Instead, we integrate the app by default.

Conveniently, this allows us to remove code modifying the .desktop file from the Linux updater (it did this to avoid asking about integration after update).

### PR Checklist

- [x] I have merged `dev` into my branch before submitting
- [ ] I have checked with either the repo owner or person who previously worked on this feature to ensure correctness (if applicable)
- [x] I have ensured that I am not using any English strings, only `t('translation_keys)`
- [x] The pull request name follows the standard commit guidelines, e.g. `feat(files): filter by file type`

### Relevant user stories or bugs

https://peerio.tpondemand.com/entity/7186

### Test plan

After releasing with this change:

* On Linux, run an older version.
* Wait for update, click "OK" when asked to update.
* After the app closes, start it again.
* Make sure the app launches without a dialog asking to integrate with the system (if this dialog appears, it's a bug, please report).

PS I tested it with my cloned repo, it works.

### SDK notes

*link to related PR in peerio-icebear*

### Gotchas/additional notes for reviewer

------

The repo owner is @anri-asaturov. The responsible QA specialist is @ebilg.

⬇️ **Note to reviewer:** please squash when merging.
